solutions/standard/variables.tf

```diff
@@ -351,12 +351,18 @@ variable "existing_code_engine_project_id" {
 
 variable "enable_kibana_dashboard" {
   type        = bool
-  description = "Set it true to deploy Kibana in code engine. NOTE: Kibana image is coming direcly from the official registry (https://www.docker.elastic.co/) and not certified by the IBM."
+  description = "Set to true to deploy Kibana in Code Engine. NOTE: By default, the Kibana image will be pulled from the official Elastic registry (docker.elastic.co) and is not certified by IBM, however this can be overridden using the `kibana_image_reference` input."
   default     = false
 }
 
 variable "elasticsearch_full_version" {
-  description = "(Optional) Full version of the Elasticsearch instance in the format `x.x.x` to deploy Kibana dashboard. If no value is passed, data lookup will fetch the full version using the Elasticsearch API, see https://github.com/elastic/kibana?tab=readme-ov-file#version-compatibility-with-elasticsearch"
+  description = "(Optional) Full version of the Elasticsearch instance in the format `x.x.x` to deploy Kibana dashboard. Value is only used if `enable_kibana_dashboard` is true and if no value is passed for `kibana_image_reference`. If no value is passed, data lookup will fetch the full version using the Elasticsearch API, see https://github.com/elastic/kibana?tab=readme-ov-file#version-compatibility-with-elasticsearch"
```

Comment:
`(Optional)` is not necessary in the description, can it be removed.

Reply:
@shemau, this is optional beacuse if the user is not providing  ES full version, data lookup will fetch the full version using the Elasticsearch API, which we have mentioned the same in the description.

Reply:
I think the point he is making is that you don't need to add the word "Optional" to the description. The rest of the description explains enough about when / why a user might want to opt into using it